src/SprykerEco/Zed/Adyen/Business/Hook/AdyenPostSaveHook.php

```diff
@@ -130,7 +130,7 @@ public function execute(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $
      */
     protected function isMethodWithRedirect(AdyenApiResponseTransfer $responseTransfer): bool
     {
-        return !empty($responseTransfer->getMakePaymentResponse()->getRedirect());
+        return (bool)$responseTransfer->getMakePaymentResponse()->getRedirect();
```

Comment:
```suggestion
        return $responseTransfer->getMakePaymentResponse()->getRedirect() !== null;
```
there is no reason to cast since `getRediredt` returns `AdyenApiMakePaymentResponseRedirect` or null

Reply:
Done.